local-tests/chainrunner/node.py

```diff
@@ -0,0 +1,92 @@
+import json
+import os.path as op
+import re
+import requests
+import jsonrpcclient as rpc
+import subprocess
+
+from .utils import flags_from_dict
+
+class node:
```

Comment:
I think that a standard way of naming classes in Python is CamelCase (and so is my strong suggestion :P)

Reply:
Yeah, you right, I thought the first letter indicates if it's supposed to be "public" or not. Confused it with Go ;)